sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko

```diff
@@ -0,0 +1,49 @@
+import { getAsObject, getAsArray, get } from '@base-cms/object-path';
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import hierarchyAliases from '@endeavorb2b/base-website-common/utils/website-section/hierarchy-aliases';
+
+$ const { site } = out.global;
+$ const section = getAsObject(data, 'section');
+$ const aliases = hierarchyAliases(section);
+$ const adSlots = {
+  'gpt-ad-lb1': getAdUnit(site, { name: 'lb1', aliases }),
+  'gpt-ad-rail1': getAdUnit(site, { name: 'rail1', aliases, size: [300, 250] }),
+  'gpt-ad-rail2': getAdUnit(site, { name: 'rail2', aliases, size: [300, 600] }),
+};
+
+<theme-pennwell-website-section-layout section=section>
+  <@head>
+    <endeavor-ad-gam-head slots=adSlots />
+  </@head>
+
+  <@above-container>
+    <endeavor-gam-ad-unit-out-of-page name="wa" aliases=aliases />
+    <endeavor-gam-ad-unit-out-of-page name="reskin" aliases=aliases />
+    <endeavor-gam-ad-unit-display id="gpt-ad-lb1" modifiers=["top-of-page"] />
+  </@above-container>
+  
+  <div class="page-wrapper default-page">
```

Comment:
The `default-page` class should be removed

Reply:
If I remove this the spacing is not set correctly, should I use margin bottom then?

Reply:
What spacing, specifically, is incorrect?

Reply:
@zarathustra323  This is what is looks like if I remove it.
![Screen Shot 2019-06-04 at 3 41 10 PM](https://user-images.githubusercontent.com/3812216/58912232-36fead80-86df-11e9-8693-00af038ed9a1.png)

Reply:
Sorry that screenshot was not the best, there is no gap between so the white from both blocks are touching.